confgenerator/resourcedetector/gce_detector.go

```diff
@@ -14,25 +14,6 @@
 
 package resourcedetector
 
-type gceAttribute int
```

Comment:
Why did you need to make this common? I don't see any new uses of this type/these constants.

Reply:
Igor proposed this because the constants are used by both the GCEResource and BMSResource structures.

Reply:
I don't see where any of these are being used in the `BMSResource` structure, can you point it out to me?

Reply:
oops, project and location constants were used by BMSResourceBuilderInterface at some point but then I made some changes and removed the interface and forgot to move the constants back to gce_detector.go.